plugins/pulp_python/plugins/migrations/0001_standard_storage_path.py

```diff
@@ -0,0 +1,43 @@
+from pulp.plugins.migration.standard_storage_path import Migration, Plan
+from pulp.server.db import connection
+
+
+def migrate(*args, **kwargs):
+    """
+    Migrate content units to use the standard storage path introduced in pulp 2.8.
+    """
+    migration = Migration()
+    migration.add(Package())
+    migration()
+
+
+class Package(Plan):
+    """
+    The migration plan for python package units.
+    """
+
+    def __init__(self):
+        """
+        Call super with collection and fields.
+        """
+        collection = connection.get_collection('units_python_package')
+        super(Package, self).__init__(collection, ('name', 'version'))
+        self.fields.add('_filename')
+
+    def _new_path(self, unit):
+        """
+        Units created by 2.8.0 don't include the filename.  This was a regression
+        that is being corrected by this additional logic.  If the storage path
+        does not end with the filename stored in the unit, it is appended.
+
+        :param unit: The unit being migrated.
+        :type unit: pulp.plugins.migration.standard_storage_path.Unit
```

Comment:
@rbarlow any idea why I'm gettting:
```
[jortel@f23d pulp_python]$ ./run-tests.py 
checking pep257 for failures, ignoring D100,D103,D104,D200,D202,D203,D205,D400,D401,D402
Function _new_path is missing sphinx type definition for argument unit (./plugins/pulp_python/plugins/migrations/0001_standard_storage_path.py:27)
found undocumented parameters!
```
I may be staring right at it and not seeing the problem.  Need a 2nd pair of eyes.